lerna.json

```diff
@@ -2,14 +2,16 @@
   "$schema": "http://json.schemastore.org/lerna",
   "version": "independent",
   "npmClient": "yarn",
-  "useWorkspaces": true,
-  "packages": [
-    "packages/*",
-    "docs"
-  ],
+  "packages": ["packages/*", "docs"],
   "command": {
     "version": {
-      "message": "chore: publish"
+      "message": "chore: publish",
+      "allowBranch": "master",
```

Comment:
If we consider to release RC/pre-releases from different branches in the future we might need to remove this rule, no?

Reply:
Yes, not remove but adapt. We could have something like `["main", "pre/*", "rc/*"]`.